media/js/devreg/reviewers/editors.js

```diff
@@ -182,7 +182,7 @@ function initReviewActions() {
 
     function check_receipt() {
         var $node = $('#receipt-check-result');
-        if ($node) {
+        if ($node.length) {
```

Comment:
`$node`, being a jQuery object, was always true, even when there is no match (`'#receipt-check-result'` only exists for paid apps). Since it doesn't exist, `$node.data('url')` returned `undefined` and we ended up doing a useless XHR call to the page we were on.